src/components/GDPRConsent.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, { useEffect } from 'react';
+
+import { GTM_ID } from '../constants';
+import CookieConsent, { getCookieConsentValue }  from "react-cookie-consent";
+import TagManager from 'react-gtm-module';
+
+
+const GDPRConsent = () => {
+  
+  const onAccept = () => {
+    // Nothing yet
+  };
+  
+  useEffect(() => {
+    // Just to ensure that when we initialize GTM, this message will be shown
+    TagManager.initialize({ gtmId: GTM_ID });
+
+    const isConsent = getCookieConsentValue();
+    if (isConsent === "true") {
+      onAccept();
+    }
+  }, []);
+
+  return (
+    <div className="cookie-consent-wrapper">
```

Comment:
Why do we need this div?

Reply:
We don't. At first i was thinking about put an overlay to prevent users interaction until hits he consent button. But i changed this behavior. I'll remove.

Reply:
Ok. Thanks!